src/images/close_black.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For simple shapes, you can try to find standard font symbols https://www.w3schools.com/charsets/ref_utf_symbols.asp